distribution/lib/Standard/Base/0.0.0-dev/src/Data/Index_Sub_Range.enso

```diff
@@ -120,7 +120,7 @@ sort_and_merge_ranges ranges =
     if sorted.is_empty then [] else
         current_ref = Ref.new sorted.first
         builder = Vector.new_builder
-        sorted.tail.each range->
+        sorted.drop.each range->
```

Comment:
I'd be rather against using this default argument scenario in our library code.

My view is - we need to have as many default arguments as possible so that the component browser can display nice previews to the user.

But I would still say there are defaults which are to be kept default and there are some that are there only because of the Component Browser requirement (because of that, ideally I think it would be neat to somehow separate these - maybe ones that are not defaulted because of the API design but because of CB previews could be provided externally? Like in a similar way to annotations for CB tagValues? Then the CB can still use them, but they would not really be defaults).

That discussion aside, I don't think `vec.drop` conveys well what is happening - ofc once we use it a lot we will start remembering its dropping one element from the vector. But IMO `(vec.drop 1)` will just be clearer.